voicebox/encryption.py

```diff
@@ -0,0 +1,236 @@
+import logging
+
+import secrets
+
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.primitives import serialization, hashes
+from cryptography.hazmat.primitives.asymmetric import padding
+from cryptography.hazmat.primitives.ciphers import (
+    Cipher,
+    algorithms,
+    modes
+)
+from cryptography.hazmat.backends import default_backend
+
+
+class BaseEncryptor:
+
+    KEY_EXCHANGE_SIGNAL = 900
+
+    def __init__(self, *args, **kwargs):
+
+        raise NotImplementedError("Cannot Initialize from BaseEncryptor")
+
+    def encrypt(self, payload: bytes):
+
+        raise NotImplementedError()
+
+    def decrypt(self, packet: bytes):
+
+        raise NotImplementedError()
+
+    def hash(self, payload):
```

Comment:
The `BaseEncryptor` class has a `hash` method that raises a `NotImplementedError`. However, this method is not used anywhere in the PR. If it's not needed, consider removing it to reduce unnecessary code. [medium]